### PR TITLE
Authentication Dialog: Display username if embedded in URL

### PR DESCRIPTION
**Pull request by linquize**
Authentication Dialog: Display username if embedded in remote URL

For example: `git remote add origin you@example.com`

Then it should fill username `you` in the Dialog, you only need to type the password.

### CODE REVIEW

File: test/spec.credentials-helper.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ describe('credentials-helper', function () {
 			server.close();
 			done();
 		});
```
Comment (codingtwinky) [inline test/spec.credentials-helper.js]: Shouldn't there be a test with and without the username embedded in url?

**Comment by linquize** [comment]
Is current `test/spec.credentials-helper.js` enough?

**Comment by codingtwinky** [comment]
But using it without user name embedded in url is still an option right?  Wouldn't that give us different code path and better coverage?

**Comment by FredrikNoren** [comment]
Closing due to inactivity

**Pull request closed without merging**